internal/cli/root.go

```diff
@@ -219,7 +265,7 @@ func (c *Client) NewCommand() *cobra.Command {
 	rootCmd.PersistentFlags().StringVar(&c.sortBy, "sort-by", "", "Sort fields for use in 'get' actions. Sort is not supported by all resources.")
 	rootCmd.PersistentFlags().StringVar(&c.sortDir, "sort-dir", "", "Sort field direction for use in 'get' actions. Sort is not supported by all resources.")
 
-	rootCmd.Version = c.Version
+	rootCmd.Version = getMetalVersion()
```

Comment:
This should still be `c.Version`; the `rootCmd.Version` is the version number for metal-cli, not for the library used by metal-cli.

Reply:
updated